platform/disk/formatter_interface.go

```diff
@@ -3,12 +3,15 @@ package disk
 type FileSystemType string
 
 const (
-	FileSystemSwap    FileSystemType = "swap"
-	FileSystemExt4    FileSystemType = "ext4"
-	FileSystemXFS     FileSystemType = "xfs"
-	FileSystemDefault FileSystemType = ""
+	FileSystemSwap             FileSystemType = "swap"
+	FileSystemExt4             FileSystemType = "ext4"
+	FileSystemXFS              FileSystemType = "xfs"
+	FileSystemDefault          FileSystemType = ""
+	FileSystemExtResizeUtility                = "resize2fs"
```

Comment:
Just add a newline between these to prevent the larger change, and separate the filesystem from the utilities.